src/main/plugin/iso19139.ca.HNAP/schematron/schematron-rules-common.sch

```diff
@@ -403,6 +403,14 @@
 
     </sch:rule>
 
+    <sch:rule context="//gmd:distributionInfo/gmd:MD_Distribution/gmd:transferOptions/gmd:MD_DigitalTransferOptions/gmd:onLine/gmd:CI_OnlineResource">
+        <sch:let name="protocolMissing" value="not(string(gmd:protocol/gco:CharacterString))" />
+    		<sch:let name="isValidProtocol" value="(string(gmd:protocol/gco:CharacterString) != 'WWW:DOWNLOAD-1.0-http--download')"/>
```

Comment:
Maybe to validate these values if are the only ones allowed? 

https://github.com/metadata101/iso19139.ca.HNAP/blob/5da459ca3a777956d3b3cf6bf7d05186711ac500/src/main/plugin/iso19139.ca.HNAP/loc/eng/labels.xml#L1751-L1756

I'm not sure if from schematron file can access the `labels.xml` file, but otherwise maybe as are few values, hardcode them in the rule?

Reply:
I cannot find any way to do it from labels.xml. So have to hardcode it. I also updated the error message to indicate

![image](https://user-images.githubusercontent.com/74916635/160003108-06ae9f40-48b1-43e8-8638-8c5f6ed7a17e.png)


But in reality, this is from a selection box and there is no way user can input some "bad" selection.